### DISCUSSION TITLE
Cann't open history experiments. Only the running one can be clicked.

### DISCUSSION THREAD

**Discussion by maminzhao** [microsoft/nni#3754] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/10074630/114729758-dc224c80-9d72-11eb-8ab3-2ea722e18320.png)

**Comment by kvartet** [comment, marked as answer]
This is by design. Users can only view the WebUI of the unstopped experiments.
You can use [nnictl resume](https://nni.readthedocs.io/en/latest/Tutorial/Nnictl.html#resume) or [nnictl view](https://nni.readthedocs.io/en/latest/Tutorial/Nnictl.html#view) to resume/view the experiments and open the WebUI URL to view them.
